continuous_integration/environment-mindeps-optional.yaml

```
@@ -49,7 +49,7 @@ dependencies:
   - zarr=2.12.0
   - pip
   - pip:
-      # optional dependencies pulled in by pip install dask[distributed]
+      # optional dependencies pulled in by pip install  dask[distributed]
```

Was this on purpose?
```suggestion
      # optional dependencies pulled in by pip install dask[distributed]
```